src/vuex/assets.module.js

```diff
@@ -101,8 +101,7 @@ export const getters = {
       .map(item => item.asset)
       .filter(item => item.isBaseInAtomicSwap),
   [vuexTypes.quoteAtomicSwapBalancesAssets]: (a, getters, b, rootGetters) =>
-    rootGetters[vuexTypes.accountBalances]
-      .map(item => item.asset)
+    rootGetters[vuexTypes.assets]
```

Comment:
the getter named `quoteAtomicSwapBalancesAssets` and is not up to date right now. pls update the getter name accordingly